huaweicloud/services/dli/resource_huaweicloud_dli_datasource_auth.go

```diff
@@ -71,7 +71,7 @@ func ResourceDatasourceAuth() *schema.Resource {
 				Sensitive:   true,
 				Description: `The password for accessing the security cluster or datasource.`,
 				RequiredWith: []string{
-					"username",
+					"user_name",
 				},
```

Comment:
the required relationship will bring an error if the users both using `username` and `password`.
please remove this relationship

Reply:
fixed